jira2markdown/markup/images.py

```diff
@@ -32,7 +32,7 @@ def action(self, tokens: ParseResults) -> str:
             if name in self.ALLOWED_ATTRS
         )
         if attrs_str:
-            return f'<img src="{tokens.url}" {attrs_str} />'
+            return f"![{tokens.url}]({tokens.url})" + "{" + attrs_str + "}"
```

Comment:
`attrs_str` should not contain a value in quotes. I just tested image attributes in the YouTrack. The following image has been render with the correct image size:
```markdown
![Markdown logo](https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/208px-Markdown-mark.svg.png "Markdown"){width=100}
```
But if attribute value is surrounded with quote, then image size doesn't change
```markdon
![Markdown logo](https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/208px-Markdown-mark.svg.png "Markdown"){width="100"}
```
You can change this behavior by replacing `f'{name}="{value}"'` a few lines above with the `f"{name}={value}"`. Related tests should be updated as well.